lib/ask/quota_bucket.ex

```diff
@@ -50,6 +50,28 @@ defmodule Ask.QuotaBucket do
     end
   end
 
+  def matches_condition?(value, [from, nil]) do
+    from = if is_integer(from), do: from, else: String.to_integer(from)
```

Comment:
**nitpick:** maybe leverage guards?

```elixir
def matches_condition?(value, [from, nil]) when is_integer(from) do
  # ...
end

def matches_condition?(value, [from, nil]) do
  matches_condition?(value, [String.to_integer(from), nil])
end
```

Reply:
I though about it, but I decided not to. 
IMHO in this particular case is clearer to have it this way. The code needed for converting the param to int is "small" and does not pollute the main logic. I think is more succinct to have the guards to identify _actual_ logic differences in this case. 
But I would love to read you if you have a strong opinion why not to 🤓

Reply:
No strong opinion, it's a nitpick (that has zero impact on performance), but it feels more elixir/erlang-y that way, and... the function itself already does just that :shrug:

```elixir
def matches_condition?(value, [from, to]) when is_integer(from) and is_integer(to) do
  # ...
end

def matches_condition?(value, [from, to]) do
  matches_condition?(value, [String.to_integer(from), String.to_integer(to)])
end
```